deepinv/training_utils.py

```diff
@@ -254,23 +296,32 @@ def train(
                         "From top to bottom : Input, Backprojection, Output, Target"
                     )
                 else:
-                    imgs = [physics_cur.A_adjoint(y), x_net, x]
+                    if isinstance(physics_cur, torch.nn.DataParallel):
+                        back = physics_cur.module.A_adjoint(y)
```

Comment:
Can we reset physics as physic.module for once somewhere ?

Reply:
no, because of the above phenomenon: we need to keep track of whether it is a `nn.DataParallel` module and potentially re-init the module during training, which cannot be done if setting `physics_cur.module` above.